app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
I'm pretty sure this results in `@authors = 5`, though it may or may not associate 5 empty authors to the paper. What's your intention with this? To have 5 form fields show up?